ksp_plugin/pile_up.hpp

```diff
@@ -143,6 +145,9 @@ class PileUp {
   using AppendToPartTrajectory =
       void (Part::*)(Instant const&, DegreesOfFreedom<Barycentric> const&);
 
+  // TODO(egg): Document this frame.
+  using ApparentPileUp = Frame<enum class ApparentPileUpTag, NonRotating>;
```

Comment:
The axes are those of `Barycentric`. The origin is the centre of mass of the pile up. This frame is distinguished from `NonRotatingPileUp` in that it is used to hold uncorrected (apparent) coordinates given by the game, before the enforcement of conservation laws; see also `ApparentBubble`.